index.html

```diff
@@ -211,10 +211,24 @@
           <h2>LDP Containers</h2>
           <p>
             Implementations MUST support the creation and management of [[!LDP]] Containers.
-            LDP Direct Containers MUST NOT permit <code>ldp:contains</code> as their membership-predicate
-            and requests that would do so MUST fail with 409 Conflict.
+          </p>
+          <p>
+            LDP Containers MUST distinguish containment triples from membership and minimal-container triples.
+            If an implementation cannot make this distinction solely from the predicate, it MUST NOT permit
+            <code>ldp:contains</code> as the <code>membership predicate</code> for LDP Direct and Indirect
+            Containers or as a client-managed property, and requests that would do so MUST fail with 409 Conflict
+            and accompanying constraints document. Implementations that can distinguish containment triples MAY
```

Comment:
Maybe they're stored in a different graph? It would depend on the implementation. The critical point is that they're either distinguished or disallowed, but I'm happy to add some more text to the informative note.